beacon-chain/db/kv/slasher.go

```diff
@@ -6,35 +6,44 @@ import (
 	"fmt"
```

Comment:
This file simply converts methods that were `single_input -> single_output` to `multiple_inputs -> multiple_outputs` for more efficient use of bolt for batched read/writes